dts/msm8916/msm8216-samsung-r05.dts

```diff
@@ -19,4 +19,14 @@
 			i2c-address = <0x25>;
 		};
 	};
+
+	a53g {
+		model = "Samsung Galaxy A5 (SM-A500H)";
+		compatible = "samsung,a53g", "qcom,msm8916", "lk2nd,device";
+		lk2nd,match-bootloader = "A500H*";
+		lk2nd,smd-rpm-hack-opening;
+
+		qcom,msm-id = <206 0>;
```

Comment:
If, you're dropping the node from 8916 file, drop the override as well (Thinking about it, yes you should only need this node unless the bootloader is funny)

Also please squash both commits into a single one